docusaurus/src/remark/docsHeaderDecoration.js

```diff
@@ -0,0 +1,135 @@
+const fetch = require("node-fetch");
+const visit = require("unist-util-visit");
+
+const registry_url =
+  "https://connectors.airbyte.com/files/generated_reports/connector_registry_report.json";
+let registry = {};
```

Comment:
alternative an unessesary idea: Instead of this we could use lodash's [memoize](https://lodash.com/docs/4.17.15#memoize) or [throttle](https://lodash.com/docs/4.17.15#throttle)

Reply:
@bnchrch I don't think memoizing or throttling is necessary here. It feels that most of the code here anyway could be shortened away, see my comment in https://github.com/airbytehq/airbyte/pull/32334/files#r1389699474 as well.